win32ss/user/ntuser/winpos.c

```diff
@@ -2648,7 +2647,7 @@ co_WinPosShowWindow(PWND Wnd, INT Cmd)
 
       default:
          //ERR("co_WinPosShowWindow Exit Good 4\n");
-         return WasVisible;
+         return FALSE;
```

Comment:
Is this codepath also tested via the (existing) tests?

Reply:
Yes. See BEFORE and AFTER comparison above.  `user32_apitest ShowWindow`